control-plane-operator/controllers/hostedcontrolplane/v2/controlplaneoperator/deployment.go

```diff
@@ -184,6 +185,17 @@ func (cpo *ControlPlaneOperatorOptions) adaptDeployment(cpContext component.Work
 func (cpo *ControlPlaneOperatorOptions) applyPlatformSpecificConfig(hcp *hyperv1.HostedControlPlane, deployment *appsv1.Deployment) {
 	switch hcp.Spec.Platform.Type {
 	case hyperv1.AWSPlatform:
+		if hcp.Spec.Platform.AWS != nil && hcp.Spec.Platform.AWS.ResourceTags != nil {
+			if awsutil.IsROSAHCPFromTags(hcp.Spec.Platform.AWS.ResourceTags) {
+				deployment.Spec.Template.Spec.Containers[0].Env = append(deployment.Spec.Template.Spec.Containers[0].Env,
```

Comment:
I am tracking the progress of the feature on ROSA, I have some comments although this is still in draft status :) 

I see there is a https://github.com/openshift/hypershift/blob/6bbea9f1f8eb84f33ef3a667a59498991beb064f/cmd/install/install.go#L183-L185

check using `--managed-service` on ho installation, shall we update it too to support ROSA ?

and will the code block of: 
https://github.com/openshift/hypershift/blob/6bbea9f1f8eb84f33ef3a667a59498991beb064f/cmd/install/assets/hypershift_operator.go#L510-L515

conflict with this change ?